app/src/main/java/com/azuresamples/msalnativeauthandroidkotlinsampleapp/SignUpCodeFragment.kt

```diff
@@ -67,8 +67,16 @@ class SignUpCodeFragment : Fragment() {
                         nextState = actionResult.nextState
                     )
                 }
-                is SignUpResult.AttributesRequired,
+                is SignUpResult.AttributesRequired -> {
+                    // Please double-check the user flow settings to ensure that the user flow is not set to collect the required attributes.
```

Comment:
I suggest using the same comment structure here:
```suggestion
                     // Please refer to the EmailAttributesSignUpFragment for handling sign up with attributes.
```